ChangeLog

```diff
@@ -42,6 +42,11 @@ Release date: TBA
 
   Closes #1085
 
+* Fixed pylint `not-callable` false positive with nested-tuple assignment in a
+for-loop.
+
+  Closes PyCQA/pylint#5113
```

Comment:
Nitpick, but could this just say "Refs" instead of closes? We will close the pylint issue with a regression test in the pylint repository.

Reply:
That's a good nitpick 👍